server/src/main/java/org/apache/druid/curator/discovery/CuratorDruidLeaderSelector.java

```diff
@@ -65,11 +63,10 @@ public CuratorDruidLeaderSelector(CuratorFramework curator, @Self DruidNode self
     this.curator = curator;
     this.self = self;
     this.latchPath = latchPath;
-
-    // Creating a LeaderLatch here allows us to query for the current leader. We will not be considered for leadership
-    // election until LeaderLatch.start() is called in registerListener(). This allows clients to observe the current
-    // leader without being involved in the election.
```

Comment:
Why has this comment been removed?

Reply:
This was by mistake. Added back with the latest commit.